modules/tinymce/changelog.txt

```diff
@@ -22,12 +22,12 @@ Version 5.3.0 (TBD)
     Fixed supplementary special characters being truncated when inserted into the editor. Patch contributed by mlitwin. #TINY-4791
     Fixed toolbar buttons not set to disabled when the editor is in readonly mode #TINY-4592
     Fixed the editor selection incorrectly changing when removing caret format containers #TINY-3438
-    Fixed bug where title, width, height would be set to empty string values when updating an image and removing those using the image dialog #TINY-4786
+    Fixed bug where title, width, or height would be set to empty string values when updating an image and removing those attributes using the image dialog #TINY-4786
```

Comment:
This should probably be `and height` instead.

Reply:
s/or/and/

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5671/commits/3afd8db191c5d5fa0ad10ebcd1665364c10dcd66